programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,10 @@
+function solution(num) {
+    let answer = '';
+    if(num % 2 == 0){
+        return "Even";
+    }
+    else {
+        return "Odd";
+    }
+    
+}
```

Comment:
변수 answer은 사용되지 않기 때문에 지워주셔도 될 것 같습니다.
```suggestion
function solution(num) {
    if (num % 2 == 0) {
        return "Even";
    }
    else {
        return "Odd";
    }
}
```

이런식으로하면 더 짧게 코드를 작성할 수 있을 것 같습니다.
```suggestion
function solution(num) {
    if (num % 2 === 0) return "Even";
    else return "Odd";
}
```